packages/wdio-reporter/src/stats/test.js

```diff
@@ -11,6 +11,7 @@ export default class TestStats extends RunnableStats {
         this.cid = test.cid
         this.title = test.title
         this.fullTitle = test.fullTitle
+        this.tags = test.tags
```

Comment:
should be here suite.tags?

Reply:
suite.tags already exists. I will remove it in test. No longer needed now.